### PR TITLE
Support skipped testcase in generated junit report

### PR DESCRIPTION
**Pull request by kingli-crypto**
Changes:
- Added `Skipped` to `TestResult`
- Emit `skipped` to skipped testcase in XML
- Added `test_ignored` to cargo test cases
- Bump version

### CODE REVIEW

**Comment by bachp** [comment]
@kingli-crypto Thank you for your contribution. Looks good.

Do you have an example of an application that produces or consumes this skipped testcases?

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "junit-report"
-version = "0.4.2"
+version = "0.4.3"
```
Comment (bachp) [inline Cargo.toml:3]: I think the change of the public enum `TestResult` makes this an API breaking change so this should be a 0.5.

**Comment by kingli-crypto** [comment]
> @kingli-crypto Thank you for your contribution. Looks good.
> 
> Do you have an example of an application that produces or consumes this skipped testcases?

I have a fork of `cargo2junit` that support `ignored` test cases from `cargo test`.
https://github.com/kingli-crypto/cargo2junit

File: Cargo.toml (hunk 1 above)
Comment (kingli-crypto) [reply Cargo.toml]: Agreed, let me update it

**Comment by kingli-crypto** [comment]
@bachp Do you think you can create a new tag for this? So it would be on crates.io

**Comment by bachp** [comment]
@kingli-crypto done

**Pull request merged**